bindings/nodejs/tests/closestSoletta.js

```diff
@@ -1,18 +0,0 @@
-var fs = require( "fs" );
-var path = require( "path" );
-var dirToCheck;
-
-// Find the closest instance of soletta via its package.json
-function closestSoletta( startDir ) {
-	for ( dirToCheck = startDir; dirToCheck !== "/";
-			dirToCheck = path.resolve( dirToCheck, ".." ) ) {
-		try {
-			if ( JSON.parse( fs.readFileSync(
-					path.join( dirToCheck, "package.json" ), "utf8" ) ).name == "soletta" ) {
```

Comment:
Here you checked if `package.json` was named `soletta`, however, new method you are using doesn't check anything like that. Is that ok? If so, r+.